docs/machineset-azure.md

```diff
@@ -9,17 +9,14 @@ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 
 `<zone>` should be replaced with a valid Azure availability zone like `us-east-1a`.
 
-`<image>` should be a WindowsServer image offering that defines the 2019-Datacenter-with-Containers SKU with version
-          17763.1457.2009030514 or earlier. Run the following command to list Azure image info:
+`<image>` should be a WindowsServer image offering that defines the 2022-Datacenter SKU.
```

Comment:
Shouldn't we recommend `2022-datacenter-smalldisk` as we are using it the machineSet.sh script and e2e test suite.